source/expressions.tex

```diff
@@ -4532,6 +4532,15 @@
 constant expression).
 \end{example}
 
+\pnum
```

Comment:
Should this be part of paragraph 6 as directed by the paper?

Reply:
First, the editing instructions in the incoming paper do say "change p6", but the actual layout of the paper text suggests that the text should actually be in a new paragraph.
Second, p6 (actually p7, it seems) talks about (really-present) _constant-expression_s and _expression_s in the _new-type-id_, whereas the new sentence talks about the absence of an expression.
So, actually I think the right fix is to move the new text slightly earlier, before we start talking about _expression_.

Reply:
Add-on patch (totally optional) pushed; @burblebee, what do you think?

Reply:
I like it :).